src/Stripe.net/Entities/Charges/ChargePaymentMethodDetailsCard.cs

```diff
@@ -36,13 +36,13 @@ public class ChargePaymentMethodDetailsCard : StripeEntity<ChargePaymentMethodDe
         /// Two-digit number representing the card’s expiration month.
         /// </summary>
         [JsonProperty("exp_month")]
-        public long ExpMonth { get; set; }
+        public long? ExpMonth { get; set; }
```

Comment:
I don't understand how that one is possible. We should ask the relevant product team what is causing this